cypress/support/commands.js

```diff
@@ -23,3 +25,8 @@
 //
 // -- This is will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+Cypress.Commands.add("login", (email, password) => {
```

Comment:
Nice adding a global command! the next step would be changing this to be an API call only and storing the cookie or jwt, but this is okay since the backend and frontend changes are out of scope for now